.github/workflows/keepalive.yml

```diff
@@ -0,0 +1,42 @@
+name: Keep scheduled workflows alive
+
+on:
+  schedule:
+    - cron: "16 07 10 * *"
+  workflow_dispatch:
+
+jobs:
+  job:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          persist-credentials: 'true'
+      - name: Create some data
+        run: |
+          mkdir -p .github/cached
+          file='.github/cached/keepalive.txt'
+          date > $file
+          git add "$file"
+      - name: Commit & push
```

Comment:
How about combining these two steps? I do not see much value in keeping them separate.